doc/sphinx/language/core/records.rst

```diff
@@ -305,17 +311,16 @@ from folded ones.
 Compatibility Projections and :g:`match`
 ++++++++++++++++++++++++++++++++++++++++
 
-To ease compatibility with ordinary record types, each primitive
-projection is also defined as an ordinary :term:`constant` taking parameters and
-an object of the record type as arguments, and whose :term:`body` is an
-application of the unfolded primitive projection of the same name. These
-constants are used when elaborating partial applications of the
-projection. One can distinguish them from applications of the primitive
-projection if the :flag:`Printing Primitive Projection Parameters` flag
-is off: For a primitive projection application, parameters are printed
-as underscores while for the compatibility projections they are printed
-as usual.
-
-Additionally, user-written :g:`match` constructs on primitive records
-are desugared into substitution of the projections, they cannot be
-printed back as :g:`match` constructs.
+To ease compatibility with ordinary record types, each primitive projection is
+also defined as an ordinary :term:`constant` taking parameters and an object of
+the record type as arguments, and whose :term:`body` is an application of the
+unfolded primitive projection of the same name. These constants are used when
+elaborating partial applications of the projection. One can distinguish them
+from applications of the primitive projection if the :flag:`Printing Primitive
+Projection Parameters` flag is off: For a primitive projection application,
+parameters are printed as underscores while for the compatibility projections
+they are printed as usual.
+
+Additionally, user-written :g:`match` constructs on primitive records are
+desugared into substitution of the projections, they cannot be printed back as
+:g:`match` constructs.
```

Comment:
This seems to be only a reformatting without a user-visible wording change.  While it's reasonable to split really long lines, reformatting in this case makes for more work in the review (i.e., figuring out whether there is a wording change) and makes it harder to identify the commits that made substantive changes.  Better not to change this now.

Reply:
I have not changed the wording here, only the formatting. I should have perhaps done the formatting in a later commit and squashed after review.

Reply:
The reason I have done these, is because I was splitting up a large line. In my IDE (VSCode) I just press a shortcut and it hard wraps the entire paragraph. I'll try to keep the diff clearer next time.

Reply:
> I have not changed the wording here, only the formatting. I should have perhaps done the formatting in a later commit and squashed after review.

Slightly better, but I would not reformat even in a separate commit absent a good reason to do so.  (And the principle applies even more strongly to source code.)